.jenkins/caffe2/build.sh

```diff
@@ -271,4 +271,11 @@ fi
 # Install ONNX into a local directory
 pip install --user -b /tmp/pip_install_onnx "file://${ROOT_DIR}/third_party/onnx#egg=onnx"
 
+if [[ $BUILD_ENVIRONMENT == *rocm* ]]; then
+  if [ -e /opt/rocm/hcc/bin/clang-7.0_original ]; then
```

Comment:
Hmm this will not work once clang version is updated, can we do it in more general way (like globing `clang-*_original`)